http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/RuntimeRoutes.scala

```diff
@@ -314,7 +314,8 @@ object RuntimeRoutes {
                                                 workerDiskSize,
                                                 numberOfWorkerLocalSSDs,
                                                 numberOfPreemptibleWorkers,
-                                                properties)
+                                                properties,
+                                                region)
```

Comment:
Bug number 2: `region` was not being passed to `RuntimeConfigRequest.DataprocConfig` here (causing it to use the default).